jme3-core/src/main/java/com/jme3/app/SimpleApplication.java

```diff
@@ -197,6 +198,9 @@ protected BitmapFont loadGuiFont() {
     public void initialize() {
         super.initialize();
 
+        SceneGraphThreadWarden.setup(rootNode);
+        SceneGraphThreadWarden.setup(guiNode);
```

Comment:
Can you make these calls with assert?  Not a big deal, but since we already 90% there, it would be ideal if the entire code was isolated (and so, supposedly optimized away) when assertions are off